api/src/main/java/org/pmiops/workbench/utils/mappers/FeaturedWorkspaceMapper.java

```diff
@@ -0,0 +1,28 @@
+package org.pmiops.workbench.utils.mappers;
+
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.pmiops.workbench.db.model.DbFeaturedWorkspace;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.model.PublishWorkspaceRequest;
+
+@Mapper(config = MapStructConfig.class)
+public interface FeaturedWorkspaceMapper {
+
+  @Mapping(target = "workspace", ignore = true)
```

Comment:
a cleaner way to do this is by `@Mapping(target = "workspace", source = "dbWorkspace")` - then you can remove the `@AfterMapping`